### PR TITLE
Upgrade `react-native-track-player` to `3.2.0`

### PR DESCRIPTION
**Pull request by zachbharris**
## Issue

Older versions of `react-native-track-player` utilize `ExoPlayer`, which has now been deprecated.

## Solution

Upgrade RNTP to their latest release `3.2.0` that does not rely on `ExoPlayer`

## Tasks

- [x] upgrade package
- [ ] successfully build with packager

### CODE REVIEW

**Comment by zachbharris** [comment]
[Update or replace react-native-track-player in audio player so that builds will succeed](https://trello.com/c/BYaYL73t/821-update-or-replace-react-native-track-player-in-audio-player-so-that-builds-will-succeed)

File: src/components/AudioPlayer/AudioPlayer.js [hunk 1]
```diff
@@ -23,7 +27,9 @@ class AudioPlayerSub extends Component {
 
     TrackPlayer.updateOptions({
       // Whether the player should stop running when the app is closed on Android
-      stopWithApp: true,
+      android: {
+        appKilledPlaybackBehavior: AppKilledPlaybackBehavior.ContinuePlayback,
+      },
```
Comment (tobyjsullivan) [inline src/components/AudioPlayer/AudioPlayer.js:32]: Is this the correct equivalent?

> stopWithApp turns into stoppingAppPausesPlayback

https://github.com/doublesymmetry/react-native-track-player/blob/main/CHANGELOG.md#300-110822

Comment (zachbharris) [reply src/components/AudioPlayer/AudioPlayer.js:32]: Probably not, I had just followed their migration guide and wanted to get straight to testing builds. I can change it now though.

**Comment by danicunhac** [comment]
PR no longer needed, so I'm closing it. @zachbharris feel free to reopen this one if there's any other work

**Pull request closed without merging**